comps/lvms_predictionguard/__init__.py

```diff
@@ -0,0 +1,2 @@
+# Copyright (C) 2024 Intel Corporation
```

Comment:
Let's change all the copyright to Prediction Guard, Inc. I think that is appropriate?

Reply:
fixed